### PR TITLE
Improve upgrade documentation

### PR DESCRIPTION
**Pull request by rgugliel-da**
Once approved, I'll add to 2.10, 2.8 and 3.x

### CODE REVIEW

File: docs/2.9.5/docs/canton/usermanual/upgrading.rst [hunk 1]
```diff
@@ -41,8 +41,15 @@ differ and will be covered separately if necessary.
 
     Upgrading requires care and preparation.
       * **Please back up your data before any upgrade.**
+
+        Note that :ref:`order of the backups <order-of-backups>` is important
```
Comment (carrielaben-da) [inline docs/2.9.5/docs/canton/usermanual/upgrading.rst:45]:
```suggestion
        Note that the :ref:`order of the backups <order-of-backups>` is important
```

File: docs/2.9.5/docs/canton/usermanual/upgrading.rst [hunk 2]
```diff
@@ -41,8 +41,15 @@ differ and will be covered separately if necessary.
...
+
       * **Please test your upgrade thoroughly before attempting to upgrade your production system.**
 
+      * **Once a migration has started it cannot be aborted or rolled back.**
+
+        In case of issue, you need to restore the old version using the backups.
```
Comment (carrielaben-da) [inline docs/2.9.5/docs/canton/usermanual/upgrading.rst:51]:
```suggestion
        If you encounter an issue, you must restore the old version using the backups.
```

**Review by carrielaben-da — APPROVED** [review]
Small changes

**Review by yves-da — APPROVED** [review]
LGTM, thanks!

Please apply the suggestions from Carrie 👍

**Pull request merged**